site/content/installation/expose-nginx-gateway-fabric.md

```diff
@@ -1,59 +1,32 @@
 ---
 title: "Expose NGINX Gateway Fabric"
 weight: 300
-toc: true
 docs: "DOCS-1427"
 ---
 
-## Overview
```

Comment:
@ADubhlaoich Just added the Overview sections to all docs, so let's not remove that

Reply:
This doc will just be embedded in the two installation methods

Reply:
I see